### PR TITLE
Prepare the release for Azure.ResourceManager.LoadTesting 1.0.0-beta.2

### LINKED ISSUE #33311: Release Azure.ResourceManager.LoadTesting 1.0.0-beta.2

### PR DESCRIPTION
**Pull request by ArthurMa1978**
fix https://github.com/Azure/azure-sdk-for-net/issues/33311

### CODE REVIEW

File: sdk/loadtestservice/Azure.ResourceManager.LoadTesting/CHANGELOG.md [hunk 1]
```diff
@@ -1,15 +1,20 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2023-01-06)
 
 ### Features Added
 
+- Update to the latest api-version.
```
Comment (Yao725) [inline sdk/loadtestservice/Azure.ResourceManager.LoadTesting/CHANGELOG.md:7]: Did we update the API version? It seems that the beta.1 has already used the API version `2022-12-01`

**Review by ArcturusZhang — APPROVED** [review]

File: sdk/loadtestservice/Azure.ResourceManager.LoadTesting/CHANGELOG.md [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
+
 ### Breaking Changes
 
 ### Bugs Fixed
 
 ### Other Changes
 
+- Update the Readme with samples.
+- Update test cases.
```
Comment (Yao725) [inline sdk/loadtestservice/Azure.ResourceManager.LoadTesting/CHANGELOG.md:16]: Maybe we should not show the test changes here as the release will not include the test project.

File: sdk/loadtestservice/Azure.ResourceManager.LoadTesting/CHANGELOG.md (hunk 1 above)
Comment (ArthurMa1978) [reply sdk/loadtestservice/Azure.ResourceManager.LoadTesting/CHANGELOG.md:7]: Yes, service team has already deployed the stable version for public preview

File: sdk/loadtestservice/Azure.ResourceManager.LoadTesting/CHANGELOG.md (hunk 2 above)
Comment (ArthurMa1978) [reply sdk/loadtestservice/Azure.ResourceManager.LoadTesting/CHANGELOG.md]: removed

**Pull request merged**